applications/GeoMechanicsApplication/tests/cpp_tests/test_process_factory.cpp

```diff
@@ -0,0 +1,123 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#include "testing/testing.h"
+#include "custom_utilities/process_factory.hpp"
+
+using namespace Kratos;
+
+
+namespace
+{
+
+class FooProcess : public Process
+{
+};
+
+class BarProcess : public Process
+{
+};
+
+}
+
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CreateNothingWhenNoCreatorWasAddedForRequestedProcess, KratosGeoMechanicsFastSuite)
+{
+    ProcessFactory factory;
+
+    const Parameters process_settings;
+    const auto process = factory.Create("UnknownProcess", process_settings);
+
+    KRATOS_EXPECT_FALSE(process)
```

Comment:
Minor suggestion: for readability, I think it would be clearer to explicitly say  KRATOS_EXPECT_TRUE(process == nullptr), because from the const auto, it's not directly clear a pointer is returned from 'Create')

Reply:
I see your point. I would, however, slightly modify your suggestion and write it as `KRATOS_EXPECT_EQ(process.get(), nullptr);`. For consistency, I will also propagate this change to some of the other unit tests in this file.